packages/server/src/distributed/base/subservice.ts

```diff
@@ -4,6 +4,8 @@ export interface DistributedSubservice {
   setup(): Promise<void>
   destroy(): Promise<void>
   listen(channel: string, callback: (message: any) => Promise<void>): Promise<void>
```

Comment:
Why not use subscribe here? It would make more sense considering that there is a function called unsubscribe that removes the listener.

Reply:
Good point changed it to subscribe